docs/xml/collection-xmldata.xml

```diff
@@ -514,14 +514,23 @@
 						Each <code>&lt;lang/&gt;</code> node may have a <literal>percentage</literal> property, which describes the percentage value to which
 						a component has been translated.
 					</para>
+					<para>
+						The source locale for a component is assumed to be <code>en_US</code>. This is the language that the untranslated strings in the
+						component are written in. In GNU Gettext terminology, this is the <code>C</code> locale. If the component has a different source
+						language, it should be specified using the <code>source-locale</code> attribute on the <code>&lt;languages/&gt;</code> element.
```

Comment:
We should use snake_case instead of kebap-case for the new property, so `source_locale` please. This is simply because the rest of the specification uses snake_case for properties and tag names, and consistency is important :-)

Reply:
Done